homeassistant/components/shopping_list/strings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "config": {
+        "title": "Shopping List",
+        "step": {
+          "user": {
+            "title": "Shopping List",
+            "description": "Do you want to configure Shopping List?"
+          }
+        },
+        "abort": {
+          "single_instance_allowed": "Only a single configuration of Shopping List is allowed."
```

Comment:
After changing to `_abort_if_unique_id_configured()`

The key will be `already_configured`